assemble/conf/accumulo-env.sh

```diff
@@ -142,5 +142,10 @@ esac
 # Variables that are optional. Uncomment to set
 ###############################################
 
-## Specifies command that will be placed before calls to Java in accumulo script
-# export ACCUMULO_JAVA_PREFIX=""
+## ACCUMULO_JAVA_PREFIX can be used to specify commands to prepend to the "java"
+## command when it is executed. This can be declared as either a scalar or an
+## array variable. The following use of declare to check if it's already set is
+## not strictly necessary, but ensures that if you set it in the calling
+## environment, that will override what is set here, rather than some mangled
+## merged result. You can set the variable any way you like.
+#declare -p 'ACCUMULO_JAVA_PREFIX' &>/dev/null || ACCUMULO_JAVA_PREFIX=''
```

Comment:
It might be useful to provide examples of how to set it in both the scalar and array manner. This may help those that are not bash experts. I'm thinking of:
```
declare -p 'ACCUMULO_JAVA_PREFIX' &>/dev/null || ACCUMULO_JAVA_PREFIX="/usr/local/bin/my_chroot_script.sh"

or

declare -p 'ACCUMULO_JAVA_PREFIX' &>/dev/null || ACCUMULO_JAVA_PREFIX=(
  'chroot'
  '/some/safe/dir'
)
```

Reply:
I feel like if somebody is editing a bash script and doesn't know bash, they can Google that... I don't think we need to be teaching bash inside our config file. It's already quite verbose in there, and there are already other examples of setting scalar and array variables inside the same file, as `var=(a b c)` and `var='abc'`, so I don't think it's really necessary.

Reply:
Does my array example above need '&&' ? If so, it might be useful to at least point out that depending on what is put in the array, you might need '&&'.

Reply:
> Does my array example above need '&&' ? If so, it might be useful to at least point out that depending on what is put in the array, you might need '&&'.

No, it doesn't. The array elements are space delimited, and elements can be quoted, but newlines are collapsed as spaces. You can run shellcheck, or shfmt, to see what they suggest, if you're not sure the optimal formatting. I also find a simple `for x in "${myarray[@]}"; do echo "$x"; done` loop is also useful for verifying behavior.

Reply:
If I had
```
declare -p 'ACCUMULO_JAVA_PREFIX' &>/dev/null || ACCUMULO_JAVA_PREFIX=(
  'chroot'
  '/some/safe/dir'
)
```

I think that would end up being `chroot /some/safe/dir /usr/bin/java ...` in the `accumulo` script, right? I don't think that will work, I think it needs to be `chroot /some/safe/dir && /usr/bin/java ...`

Reply:
Yeah, but that's not what this is for. This isn't to run arbitrary code prior to executing Java. It's specifically to wrap the Java command with some executor, like `numactl`, `strace`, `time`, or similar. If you want to switch directories or run other command *before* you run the java command, you'll have to do that inside your env script, or outside the call to `bin/accumulo`.

Reply:
ok, so my example was incorrect

Reply:
> ok, so my example was incorrect

Yeah, sorry, I wasn't paying attention to the actual command you were trying to run... I interpreted them as arbitrary examples. If you try to pass `(chroot path "&&")`, I'm not sure what bash will do. It may do what you want, or it may pass "&&" as an argument to chroot.